### ISSUE TITLE
pdf download 啊

### ISSUE THREAD

**Issue by gigberg** [gdh1995/pdf.js#2]
Attach (recommended) or Link to PDF file here:

Configuration:
- Web browser and its version:
- Operating system and its version:
- PDF.js version:
- Is a browser extension:

Steps to reproduce the problem:
1. 
2.

What is the expected behavior? (add screenshot)

What went wrong? (add screenshot)

Link to a viewer (if hosted on a site other than mozilla.github.io/pdf.js or as Firefox/Chrome extension):

**Comment by gigberg** [comment]
Attach (recommended) or Link to PDF file here: 
https://www.ccf.org.cn/ccf/contentcore/resource/download?ID=101401
https://winprotocoldoc.blob.core.windows.net/productionwindowsarchives/MS-RDPADRV/%5BMS-RDPADRV%5D-210407.pdf

Web browser and its version: Chrome Version 100.0.4896.75 (Official Build) (32-bit)
PDF.js version: installed from google webstore
Steps to reproduce the problem:
1. enable the extension
2. browse the url

What is the expected behavior? (add screenshot)
the pdf open in new tab without download
What went wrong? (add screenshot)
the extension works for most pdf link, but sometimes (the urls above) pdf files is download by chrome. However, the "pdf viewer" extension provide by Molliza in crome webstore still works well in these urls.

pdf viewer for vimium c: https://chrome.google.com/webstore/detail/pdf-viewer-for-vimium-c/nacjakoppgmdcpemlfnfegmlhipddanj
pdf viewer by Molliza: https://chrome.google.com/webstore/detail/pdf-viewer/oemmndcbldboiebfnladdacbdfmadadm

**Comment by gdh1995** [comment]
Yes the feature has some limitation and can not be fixed easily. Please turn off it if you need.

**Comment by gdh1995** [comment]
It seems this single issue can be fixed. I'll take a try.

**Comment by gigberg** [comment]
> It seems this single issue can be fixed. I'll take a try.

Nice job! Could you please updated this in chrome webstore.

**Comment by gdh1995** [comment]
Updated and you may take a try.

**Comment by gigberg** [comment]
> Updated and you may take a try.

cannot use  back or forward after checking "Prefer keeping URLs in the address bar unchange"
https://github.com/mozilla/pdf.js/issues/10360
